blueprints/wpcli-markdown/blueprint.json

```
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"meta": {
+		"title": "Content via wp-cli and markdown",
+		"description": "create post from markdown file with an image",
+		"author": "bph",
+		"categories": ["Content", "wpcli"]
+	},
+	"landingPage": "/wp-admin/",
+	"steps": [
+		{
+			"step": "login"
+		},
+		{
+			"step": "writeFile",
+			"path": "/wp-content/uploads/postcontent.md",
+			"data": {
+				"resource": "url",
+				"url": "https://gutenbergtimes.com/wp-content/uploads/2020/11/wpcli-markdown/postcontent.md"
+			}
+		},
+		{
+			"step": "wp-cli",
+			"command": "wp post create --post_title 'Welcome to Playground'  /wp-content/uploads/postcontent.md"
+		}
+	]
+}
```

Two problems here:

* It's `/wordpress/wp-content`, not `/wp-content`.
* `/wp-content/uploads` doesn't exist by default in WordPress, I updated the path to just `/wp-content/postcontent.md`

```suggestion
{
  "$schema": "https://playground.wordpress.net/blueprint-schema.json",
  "meta": {
    "title": "Content via wp-cli and markdown",
    "description": "create post from markdown file with an image",
    "author": "bph",
    "categories": [
      "Content",
      "wpcli"
    ]
  },
  "landingPage": "/wp-admin/",
  "steps": [
    {
      "step": "login"
    },
    {
      "step": "writeFile",
      "path": "/wordpress/wp-content/postcontent.md",
      "data": {
        "resource": "url",
        "url": "https://gutenbergtimes.com/wp-content/uploads/2020/11/wpcli-markdown/postcontent.md"
      }
    },
    {
      "step": "wp-cli",
      "command": "wp post create --post_title='Welcome to Playground' /wordpress/wp-content/postcontent.md"
    }
  ]
}
```